local.sh

```diff
@@ -23,7 +23,6 @@ echo '### Deriving temporary config (converting mkdocs.yml to local.yml)'
 rm -f local.yml
 cat mkdocs.yml | sed \
 	-e 's#^INHERIT.*##' \
-	-e 's#^site_url.*$#site_url: ""#' \
```

Comment:
What does this change do?

Reply:
Before change: changes site_url setting made in mkdocs.yml (set to https://code.kx.com/q) to an empty string. 

This prevents sitemap.xml from being generated among other things (https://www.mkdocs.org/user-guide/configuration/)